docs/standard/linq/toc.yml

```
@@ -0,0 +1,346 @@
+      href: linq-to-xml-vs-other-xml-technologies.md
+  - name: Programming Guide (LINQ to XML)
+    items:
+    - name: LINQ to XML Programming Overview
+      items:
+      - name: Functional vs. Procedural Programming (LINQ to XML)
+        href: functional-vs-procedural-programming-linq-to-xml.md
+      - name: LINQ to XML Classes Overview
+        href: linq-to-xml-classes-overview.md
+      - name: XElement Class Overview
+        href: xelement-class-overview.md
+      - name: XAttribute Class Overview
+        href: xattribute-class-overview.md
+      - name: XDocument Class Overview
+        href: xdocument-class-overview.md
+      - name: "How to: Build LINQ to XML Examples"
+        href: how-to-build-linq-to-xml-examples.md
+    - name: XML Trees
+      items:
+      - name: Functional Construction (LINQ to XML)
+        href: functional-construction-linq-to-xml.md
+      - name: Creating XML Trees
+        href: creating-xml-trees.md
+      - name: Parsing XML
+        items:
+        - name: "How to: Parse a String"
+          href: how-to-parse-a-string.md
+        - name: "How to: Load XML from a File"
+          href: how-to-load-xml-from-a-file.md
+        - name: Preserving White Space while Loading or Parsing XML
+          href: preserving-white-space-while-loading-or-parsing-xml.md
+        - name: "How to: Catch Parsing Errors"
+          href: how-to-catch-parsing-errors.md
+        - name: "How to: Create a Tree from an XmlReader"
+          href: how-to-create-a-tree-from-an-xmlreader.md
+        - name: "How to: Stream XML Fragments from an XmlReader"
+          href: how-to-stream-xml-fragments-from-an-xmlreader.md
+      - name: "How to: Populate an XML Tree with an XmlWriter (LINQ to XML)"
+        href: how-to-populate-an-xml-tree-with-an-xmlwriter-linq-to-xml.md
+      - name: "How to: Validate Using XSD (LINQ to XML)"
+        href: how-to-validate-using-xsd-linq-to-xml.md
+      - name: Valid Content of XElement and XDocument Objects
+        href: valid-content-of-xelement-and-xdocument-objects.md
+    - name: Working with XML Namespaces
+      items:
+      - name: Namespaces Overview (LINQ to XML)
+        href: namespaces-overview-linq-to-xml.md
+      - name: "How to: Create a Document with Namespaces (LINQ to XML)"
+        href: how-to-create-a-document-with-namespaces.md
+      - name: "How to: Control Namespace Prefixes (LINQ to XML)"
+        href: how-to-control-namespace-prefixes-linq-to-xml.md
+      - name: Scope of Default Namespaces
+        href: scope-of-default-namespaces.md
+      - name: Working with Global Namespaces in Visual Basic
+        href: working-with-global-namespaces.md
+      - name: "How to: Write Queries on XML in Namespaces"
+        href: how-to-write-queries-on-xml-in-namespaces.md
+    - name: Serializing XML Trees
+      items:
+      - name: Preserving White Space While Serializing
+        href: preserving-white-space-while-serializing.md
```

let's remove ing when possible too

```suggestion
        href: preserve-white-space-while-serializing.md
```